models/stg_snapchat.yml

```diff
@@ -356,4 +368,131 @@ models:
+    description: Each record represents a daily performance summary per country for each campaign.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - source_relation
+            - campaign_id
+            - date_day
+            - country 
+    columns:
+      - name: source_relation
+        description: "{{ doc('source_relation') }}"
+      - name: campaign_id
+        description: ID of campaign
+        tests:
+          - not_null
+      - name: country
+        description: The country that the report relates to.
+        tests:
+          - not_null
+      - name: date_day
+        description: The day that the report relates to.
+        tests:
+          - not_null
+      - name: attachment_quartile_1
+        description: Number of times your Long Form Video was viewed to 25% 
+      - name: attachment_quartile_2
+        description: Number of times your Long Form Video was viewed to 50% 
+      - name: attachment_quartile_3
+        description: Number of times your Long Form Video was viewed to 75% 
+      - name: attachment_total_view_time
+        description: Total time Snapchatters spent on the attachment (HH:MM:SS)
+      - name: attachment_view_completion
+        description: The number of times your Long Form Video was viewed to completion (97%)
+      - name: quartile_1
+        description: The number of times your video was viewed to 25% 
+      - name: quartile_2
+        description: The number of times your video was viewed to 50% 
+      - name: quartile_3
+        description: The number of times your video was viewed to 75% 
+      - name: saves
+        description: Number of times components of the campaign were saved to Memories
+      - name: shares
+        description: Number of times components of the campaign were shared in a Chat or Story
+      - name: screen_time
+        description: The number of seconds spent watching your campaign across all paid impressions. Screen time starts recording as soon as the media is fully rendered on the device and the autoplay video starts
+      - name: video_views
+        description: The number of times your video was watched at least 2 seconds or swiped up, whichever comes first
+      - name: view_completion
+        description: The number of times your video was viewed to completion (97% )
+      - name: view_time
+        description: The number of seconds spent watching your campaign across all users reached.
+      - name: impressions
+        description: The number of impressions for a campaign.
+      - name: spend
+        description: The amount of spend for a campaign.
+      - name: swipes
+        description: The number of swipes for a campaign. Swipes are when your ad was swiped up on or the CTA was tapped to view the attachment below.
+      - name: conversion_purchases_value
+        description: The total value of purchases attributed to a campaign.
+      - name: conversion_purchases
```

Comment:
This field isn't brought into the staging model. Does this need to be added or should we remove this?

Reply:
Ah `conversion_purchases ` is brought in via the conversion variable [here](https://github.com/fivetran/dbt_snapchat_ads_source/blob/827d0d658d843f997104be86753219c93ac6a754/dbt_project.yml#L19)